### ISSUE TITLE
Enable Smooth Scrolling

### ISSUE THREAD

**Issue by flipflop97** [termux/termux-app#118]
It would be awesome if smooth/pixel perfect scrolling could be implemented.
With smooth scrolling I mean that when dragging the screen the content will move per pixel (the native Android scrolling) rather than per line in the terminal.

**Comment by Neo-Oli** [comment]
I don't think this is even possible with a terminal. The scrollback of a terminal isn't just a very high window that you scroll up and down on. But if you want pixel scrolling you'd need exactly that. Width and Height of a Terminal are very important to various fullscreen applications like vim and weechat and are set in $COLUMNS and $LINES. How would it deal with that? Do you know of any terminal emulator which has this feature?

**Comment by flipflop97** [comment]
Yep, gnome terminal 3.20 has this feature. It's not enabled on raw in-/output like vim or nano but otherwise it is.

**Comment by Rixon-gh** [comment]
Sorry for digging up a necropost but I'd just like to add that the main importance for me is not so much the 'pizel perfect scrolling', but the ability to 'fling' a scroll with momentum further than you could swipe on the screen. Scrolling through a long piece of output can become tedious.

**Comment by Rixon-gh** [comment]
Sorry to dig up a necropost but I just wanted to add that it's not the 'pixel perfect scrolling' that is so important to me, but the ability to 'fling' with momentum further than you could swipe on the screen - I know this is possible most of the time but some programs such as pagers like `less` or `man` seem to disable it, so scrolling through a loooong manpage can become tedious quite quickly.
